### PR TITLE
Feat: Add new dataset card

### PR DESCRIPTION
**Pull request by Jorek57**

### CODE REVIEW

File: udata_front/theme/gouvfr/datagouv-components/src/components/typography.less [hunk 1]
```diff
@@ -0,0 +1,14 @@
+@import "../../../assets/less/variables.less";
+
+.datagouv-components {
+  .dash-after-sm::after {
```
Comment (nicolaskempf57) [inline udata_front/theme/gouvfr/datagouv-components/src/components/typography.less:4]: Can you move this to the typography file in udata-front ?

File: udata_front/theme/gouvfr/datagouv-components/src/composables/useLicense.ts [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import {ref, unref} from "vue";
+import fetchLicenses from "../api/licenses";
+import { type License } from "../types/licenses";
+
+/** TODO : to be removed when the component moved to data.gouv.fr-components */
```
Comment (nicolaskempf57) [inline udata_front/theme/gouvfr/datagouv-components/src/composables/useLicense.ts:5]: Can you put this comment in udata-front instead ?

**Comment by Jorek57** [comment]
This Pr is jsut to implement the new dataset card in the datagouv-components package. The implementation of the new card  in udata-front will be done in a future PR

**Review by maudetes — APPROVED** [review]
Nice! :heart_eyes: Looks good to me :clap:

**Pull request merged**